traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -365,7 +379,16 @@ var TableServersController = function(tableName, servers, filter, $scope, $state
 		},
 		onRowClicked: function(params) {
 			const selection = window.getSelection().toString();
-			if(selection === "" || selection === $scope.mouseDownSelectionText) {
+			const overrideCells = [
```

Comment:
I'd move this to a field on the controller since it never changes.

Reply:
10-4, this was added mostly last minute to ensure that the default behavior for the row wasn't applied